lib/engines/content_object_store/test/integration/content_block_editions_test.rb

```diff
@@ -3,15 +3,86 @@
 
 class ContentBlockEditionsTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
+  extend Minitest::Spec::DSL
 
-  test "#index returns all Content Block Editions" do
-    content_block_document = create(:content_block_document)
-    create(
-      :content_block_edition,
-      details: '"email_address":"example@example.com"',
-      content_block_document_id: content_block_document.id,
-    )
-    visit "/government/admin/content-object-store/content-block-editions"
-    assert_text '"email_address":"example@example.com"'
+  before do
+    login_as_admin
+  end
+
+  describe "#index" do
+    test "it returns all Content Block Editions" do
+      content_block_document = create(:content_block_document)
+      create(
+        :content_block_edition,
+        details: '"email_address":"example@example.com"',
+        content_block_document_id: content_block_document.id,
+      )
+      visit "/government/admin/content-object-store/content-block-editions"
+      assert_text '"email_address":"example@example.com"'
+    end
+  end
+
+  describe "#new" do
+    test "it shows a list of all the valid block types" do
+      schemas = [
+        ContentObjectStore::Schema.new("content_block_foo", { "properties" => { "foo" => {}, "bar" => {} } }),
+        ContentObjectStore::Schema.new("content_block_bar", { "properties" => { "foo" => {}, "bar" => {} } }),
+      ]
+
+      given_there_are_existing_schemas schemas
```

Comment:
Whoops I just realised we already have Cucumber in the project. Will refactor these tests to be Cukes instead